src/ape_geth/providers.py

```diff
@@ -118,6 +119,9 @@ class GethNetworkConfig(PluginConfig):
     # Make sure to run via `geth --dev` (or similar)
     local: dict = DEFAULT_SETTINGS.copy()
 
+    class Config:
+        extra = Extra.allow
```

Comment:
maybe we need a comment saying what this is for? "Adding ecosystems that use Geth, such as geth-forks"

Reply:
also (talked offline) but this is the wrong spot, needs to be in the class below.

After that, things seem to work in ape-fantom!